src/main/java/net/minestom/server/utils/chunk/ChunkUtils.java

```diff
@@ -211,7 +216,9 @@ public static int getBlockIndex(int x, int y, int z) {
      */
     public static @NotNull Point getBlockPosition(int index, int chunkX, int chunkZ) {
         final int x = blockIndexToChunkPositionX(index) + Chunk.CHUNK_SIZE_X * chunkX;
-        final int y = index >>> 4 & 0xFF;
+        int y = index >>> 4 & 0xFF;
```

Comment:
i think this should be `final int y = index << 4 >> 8 & 0xFF`

Reply:
actually, why is this mask `& 0xFF` maybe this should use blockIndexToChunkPositionY and then change that to use the change i mentioned?